docs/guide/http_cache.md

```diff
@@ -117,6 +117,12 @@ as it will cause your reverse proxy to wait for back end and not be able to deli
     Note that sharing ESIs across SiteAccesses when using URI matching is not possible by design,
     as the URI contains the SiteAccess name encoded in its path information.
 
+## Symfony reverse proxy
+
+To use Symfony reverse proxy, follow the [Symfony documentation](https://symfony.com/doc/current/http_cache.html#symfony-reverse-proxy).
+
+Instead of `Symfony\Bundle\FrameworkBundle\HttpCache\HttpCache`, your caching kernel must use `\EzSystems\PlatformHttpCacheBundle\AppCache`.
```

Comment:
```suggestion
However, instead of `Symfony\Bundle\FrameworkBundle\HttpCache\HttpCache`, your caching kernel must use `\EzSystems\PlatformHttpCacheBundle\AppCache`.
```
Hopefully "however" is enough to turn the reader's attention to a discrepancy with the original instructions.